pkg/export/attributes/metric.go

```diff
@@ -101,6 +101,16 @@ var (
 		Prom:    "messaging_process_duration_seconds",
 		OTEL:    "messaging.process.duration",
 	}
+	GPUKernelLaunchCalls = Name{
+		Section: "gpu.kernel.launch.calls",
+		Prom:    "gpu_kernel_launch_calls_total",
+		OTEL:    "gpu.kernel.launch.calls",
+	}
+	GPUMemoryAllocations = Name{
```

Comment:
Is it worth to use these semantic convetions: https://opentelemetry.io/docs/specs/semconv/system/hardware-metrics/#hwgpu---gpu-metrics?

Reply:
They don't have the support for what we want to track here. Those gpu metrics are like our process metrics, what we track here are kernel launches and memory allocations. For example, if you compare to Go GC metrics, what the spec contains is memory used by the Go process, but what we track is number of objects allocated by the application.